packages/core/strapi/lib/services/entity-service/types/index.d.ts

```diff
@@ -27,4 +33,9 @@ export interface EntityService {
     uid: TContentTypeUID,
     params?: EntityService.Params.Pick<TContentTypeUID, 'data' | 'files' | 'fields' | 'populate'>
   ): Promise<Attribute.GetValues<TContentTypeUID>>;
+  update<TContentTypeUID extends Common.UID.ContentType>(
+    uid: TContentTypeUID,
+    entityId: number,
+    params?: EntityService.Params.Pick<TContentTypeUID, 'data' | 'fields' | 'populate'>
```

Comment:
```suggestion
    params?: EntityService.Params.Pick<TContentTypeUID, 'data' | 'files' | 'fields' | 'populate'>
```

Reply:
Where do you see the files param? I can't find it: https://github.com/strapi/strapi/blob/9efedc2ca2ebf393c5a8cccbf9ffe3087fb426d6/packages/core/strapi/lib/services/entity-service/params.js#L5

Reply:
Second line of the method :slightly_smiling_face:

Reply:
Does this documentation need to be updated: https://docs.strapi.io/dev-docs/api/entity-service/crud#update ?

Reply:
Seems like it should yeah